yarn.lock

```diff
@@ -1743,6 +1750,11 @@
   "resolved" "https://registry.npmjs.org/flatted/-/flatted-3.2.4.tgz"
   "version" "3.2.4"
 
+"follow-redirects@^1.14.4":
```

Comment:
Do we really need this?

Reply:
I didn't install this, I just installed axios with `npm install axios`

Reply:
Cool, I believe removing axios should remove this as well then.